src/Integration/Helpers/VsShellUtils.cs

```diff
@@ -85,26 +85,6 @@ public static void WriteToSonarLintOutputPane(IServiceProvider serviceProvider,
             }
         }
 
-        public static bool IsSolutionExistsAndNotBuildingAndNotDebugging()
-        {
-            Debug.Assert(KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext != null, "KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebugging is null");
-            if (KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext != null)
-            {
-                return KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext.IsActive;
-            }
-            return false;
-        }
-
-        public static bool IsSolutionExistsAndFullyLoaded()
-        {
-            Debug.Assert(KnownUIContexts.SolutionExistsAndFullyLoadedContext != null, "KnownUIContexts.SolutionExistsAndFullyLoadedContext is null");
-            if (KnownUIContexts.SolutionExistsAndFullyLoadedContext != null)
-            {
-                return KnownUIContexts.SolutionExistsAndFullyLoadedContext.IsActive;
-            }
-            return false;
-        }
-
```

Comment:
The binding controller used these two static helper methods that in turn accessed `KnownUIContexts`. I've dropped these methods and changed the binding controller to reference `IKnownUIContexts` directly.